Swarm/src/index.ts

```diff
@@ -79,6 +79,12 @@ class CitrineOSServer {
             initSwagger(this._config, this._server);
         }
 
+        // Add Directus Message API flow creation if enabled
+        if (this._config.util.directus?.generateFlows) {
+            const directusUtil = new DirectusUtil(this._config, this._logger);
+            this._server.addHook("onRoute", directusUtil.addDirectusMessageApiFlowsFastifyRouteHook.bind(directusUtil));
+        }
```

Comment:
From previous sessions, I think we would benefit to know when the directus flows setup finished. I looked at the docs here: https://fastify.dev/docs/latest/Reference/Hooks/#onready and it seems that Fastify has a hook when it's stood up and all routes are registered. I think that's the place to log a success. 

```suggestion
            this._server.addHook('onReady', async () => {
                this._logger.info('Directus actions initialization finished');
            });
        }
```